packages/data-row-permissions/src/components/DataStatus.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div v-if="results.length === 0">
+    <div class="align-center-container">
+      <div
+        v-if="!resultsLoaded"
+        class="align-center text-center ">
+        <b-spinner
+          label="Spinning"
+          variant="primary" />
+        <div class="mt-5">
+          {{ $t('data-row-permissions-spinner-text') }}
+        </div>
+      </div>
+      <span
+        v-else-if="isAuthenticated"
+        class="mt-2">{{ $t('data-row-permissions-no-results') }}</span>
+      <span
+        v-else
+        class="mt-2">{{ $t('data-row-permissions-no-results-cta') }}&nbsp;<a href="/login">{{ $t('data-row-permissions-cta-login') }}</a>.</span>
```

Comment:
Since this is an admin screen, why not open the login dialog right away if user isn't authenticated?

Reply:
technically it's not admin only, it's for owners of objects. So little extra hints aren't that bad?